About.html

```diff
@@ -114,33 +114,30 @@ <h3 class="partner-heading">Partner</h3>
       <hr class="heading-underline" />
       <div class="partners-list">
         <div class="img-car">
-        <img src="assets/images/partners/mozilla 1.png" alt="" class="p-image" />
-      
-        <img src="assets/images/partners/googlekorea 1.png" alt="" class="p-image" />
-      
-        <img src="assets/images/partners/naver 1.png" alt="" class="p-image" />
-      </div>
+          <img src="assets/images/partners/mozilla 1.png" alt="" class="p-image" />
+          <img src="assets/images/partners/googlekorea 1.png" alt="" class="p-image" />
+          <img src="assets/images/partners/naver 1.png" alt="" class="p-image" />
+        </div>
         <div class="img-car2">
-        <img src="assets/images/partners/daumkakaologo 1.png" alt="" class="p-image" />
-      
-        <img src="assets/images/partners/airbnb 1.png" alt="" class="p-image" />
-      </div>
+          <img src="assets/images/partners/daumkakaologo 1.png" alt="" class="p-image" />
+          <img src="assets/images/partners/airbnb 1.png" alt="" class="p-image" />
+        </div>
```

Comment:
- [ ] As mentioned by the previous reviewer, for accessibility reasons and for semantic purposes I would suggest you add the appropriate information in each `alt` attribute of your `img` tag. 
**I recommend you apply this approach to all your `img` tags of your code base**.

Reply:
> * [ ]  As mentioned by the previous reviewer, for accessibility reasons and for semantic purposes I would suggest you add the appropriate information in each `alt` attribute of your `img` tag.
>   **I recommend you apply this approach to all your `img` tags of your code base**.

- [ ] Just as the previous reviewer recommended, I would follow suite because I noticed the whole img tags in the About.html file has no alt attribute and alt attributes are not excusable because they give meaning to other devices such as screen readers so I will encourage you to add it to all, even in the index file. You can read more on alt attribute [here](https://www.w3schools.com/tags/att_img_alt.asp)